### PR TITLE
[LV-624 - Nightmare Insert] Command Dome

### PR DESCRIPTION
**Pull request by Steelpoint**
# About the pull request

This PR adds a new nightmare insert of the gym that can transform it into a 'Command Dome', the structure contains amenities expected of a command area, however it is also a new possible spawn location for a comms array. 

It uses the number 1 comms spawn, meaning it shares the spawn location with the Engineering comms spawn. Ergo there is a 50/50 the 1 comms spawns either in south Engineering, or central Command. 

This also adds a rarer alt spawn for the Command Dome that spawns it as a heavily fortified, but breached, area that some survivors used. 

# Explain why it's good for the game

The addition of a second possible colony spawn for a comms array would help make the central colony an actually desirable location to hold and fortify for both sides, and make the colony a possible combat zone.

As it stands, the central colony is rarely a fought over area as both sides normally use it as a transit zone from the FOB to the cave frontline, and any major fighting is usually contained towards the west/east rivers or Engineering, ergo Marines never really fortify the central colony thus it becomes a backline area.

This change would have a chance that both sides now have to fight over the centre of the colony. Thus breathing new life into this area of the map that is so often bypassed. 

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Main Spawn
Screenshot 2024-03-20 16 21 54

Uncommon Alt Spawn
Screenshot 2024-03-20 16 17 14

</details>

# Changelog
:cl:
mapadd: A new 'Command Dome' nightmare insert has been added to LV-624. Having a chance to appear in place of the Gym.
/:cl:

### CODE REVIEW

File: maps/map_files/LV624/LV624.dmm
Comment (Nanu308) [inline maps/map_files/LV624/LV624.dmm:1]: Keep fitness, make command instead a NM insert and work from there 👍

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/LV624/LV624.dmm
Comment (Steelpoint) [reply maps/map_files/LV624/LV624.dmm:1]: can do

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by Steelpoint** [comment]
Assuming nothing breaks I'll re-work the PR in <~1week

**Comment by Steelpoint** [comment]
I've updated this to be a nightmare insert that has a chance to replace the Gym with a Command Dome.

**Comment by Nanu308** [comment]
I am back from being dead.

After some further considerations I think the command insert would fit better into the Tcomms location on LV. Having a tcomms so close to Hydro i dont think is a good thing due to how frequent Marines entrench or go to hold Hydro.

Due to that I'd like to see if as a tcomms variant instead 👍

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by Steelpoint** [comment]
Will finish this when I have free time to fix mapping issues and make requested changes

**Pull request closed without merging**